### DISCUSSION TITLE
Checking for an empty dataframe

### DISCUSSION THREAD

**Discussion by derrickcchow** [vaexio/vaex#1859] [Q&A] [answered]
Hi,

I was wondering what is the best way to check for an empty database?

Thanks Derrick

**Comment by maartenbreddels** [comment, marked as answer]
Hi Derrick,

`len(df) == 0` is the best way! Hope that was what you meant

Regards,

Maarten

**Reply by derrickcchow** [reply]
Ya, I meant data frame and it works great thanks!
